manuscript/references.bib

```diff
@@ -1787,8 +1787,8 @@ @misc{reticulate
 
 @misc{ramlegacy,
     author = {{RAM Legacy Stock Assessment Database}},
-    title = {RAM Legacy Stock Assessment Database v4.44},
-    year = {2018},
-    doi = {10.5281/zenodo.2542919},
-    url = {https://doi.org/10.5281/zenodo.2542919},
+    title = {RAM Legacy Stock Assessment Database v4.491},
+    year = {2020},
+    doi = {10.5281/zenodo.3676088},
+    url = {http://doi.org/10.5281/zenodo.3676088},
```

Comment:
@cboettig I was having problems with this link which I ripped from Appendix D. Just thought I'd let you know if the link is dead.

Reply:
the doi URL works for me

Reply:
hmph working for me now also